zstacklib/zstacklib/test/utils/http_test_client.py

```
@@ -0,0 +1,484 @@
+        req.add_header(TASK_UUID, task_uuid)
+        req.add_header(CALLBACK_URI, callback_url)
+        req.add_header('commandpath', path)
+
+        try:
+            urllib.request.urlopen(req, timeout=5)
+        except Exception:
+            pass
+
+        # SSH-poll for callback result
+        deadline = time.monotonic() + timeout
+        cb_path = '/tmp/callbacks/%s.json' % task_uuid
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/callbacks/%s.json"

(S108)